### PR TITLE
Use prettier for code formatting

### PR DESCRIPTION
**Pull request by JuliaGerasymenko**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,7 +33,8 @@
   ],
   "scripts": {
     "test": "npm run lint && metatests test/",
-    "lint": "eslint --ignore-path .gitignore .",
+    "lint": "eslint . --ignore-path .gitignore",
```
Comment (nechaido) [inline package.json]: Is this needed?

Comment (JuliaGerasymenko) [reply package.json]: I found that in most of repositories the dot goes after eslint not at the end of a command. Alright, I'll change it back.

File: README.md [hunk 2]
```diff
@@ -17,32 +18,28 @@ $ npm install metaschema
 
 #### create(schemas)
 
-  - `schemas: `[`<Iterable>`] schemas in form [name, schema, source] (the
-        'source' is optional)
-
-*Returns:* `[ `[`<MetaschemaError>`]`, `[`<Metaschema>`]` ]`
+- `schemas:`[`<Iterable>`] schemas in form [name, schema, source] (the
+  'source' is optional)
```
Comment (SemenchenkoVitaliy) [inline README.md]: I would prefer that this line had this line had the same indentation from parameter comment beginning.

**Comment by nechaido** [comment]
@JuliaGerasymenko please add `README.md` to `.prettierignore`, don't forget to reset changes i `README.md`.

**Comment by JuliaGerasymenko** [comment]
@nechaido Should I do the same in another repositories?

**Comment by nechaido** [comment]
@JuliaGerasymenko this should be done for any autogenerated file.

File: .prettierignore [hunk 3]
```diff
@@ -0,0 +1,5 @@
+node_modules
```
Comment (belochub) [inline .prettierignore]: There is no need to add `node_modules` here since it is ignored by default.

File: .prettierignore [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
+*.log
```
Comment (belochub) [inline .prettierignore]: These files are not `.js`, `.md` or `.json`, and you run `prettier` only on files with extensions mentioned above:
https://github.com/metarhia/metaschema/blob/2256c955ad58d93374f15ffd0b104b7d5e650e0d/package.json#L36-L37
So there is no point in adding this line.
```suggestion
```

File: .prettierignore [hunk 5]
```diff
@@ -0,0 +1,5 @@
...
+.DS_Store
```
Comment (belochub) [inline .prettierignore]: Same here.
```suggestion
```

File: package.json [hunk 6]
```diff
@@ -33,7 +33,8 @@
   ],
   "scripts": {
     "test": "npm run lint && metatests test/",
-    "lint": "eslint --ignore-path .gitignore .",
+    "lint": "eslint --ignore-path .gitignore . && prettier --debug-check \"**/*.js\" \"**/*.json\" \"**/*.md\" .eslintrc .prettierrc  --ignore-path .prettierignore",
```
Comment (nechaido) [inline package.json]: `.prettierignore` is used by default
```suggestion
    "lint": "eslint --ignore-path .gitignore . && prettier --debug-check \"**/*.js\" \"**/*.json\" \"**/*.md\" .eslintrc .prettierrc",
```

File: package.json [hunk 7]
```diff
@@ -33,7 +33,8 @@
...
+    "fmt": "prettier --write \"**/*.js\" \"**/*.json\" \"**/*.md\" .eslintrc .prettierrc --ignore-path .prettierignore",
```
Comment (nechaido) [inline package.json]:
```suggestion
    "fmt": "prettier --write \"**/*.js\" \"**/*.json\" \"**/*.md\" .eslintrc .prettierrc",
```

**Pull request merged**